lambdas/services/lloyd_george_stitch_service.py

```diff
@@ -34,7 +34,8 @@ def __init__(self):
             "STITCHED_FILE_LIFECYCLE_POLICY_TAG", "autodelete"
         )
 
-        self.s3_service = S3Service()
+        create_document_aws_role_arn = os.getenv("PRE_SIGN_ASSUME_ROLE")
+        self.s3_service = S3Service(custom_aws_role=create_document_aws_role_arn)
```

Comment:
```suggestion
       presign_url_aws_role_arn = os.getenv("PRE_SIGN_ASSUME_ROLE")
        self.s3_service = S3Service(custom_aws_role=presign_url_aws_role_arn)
```
suggest to rename as here the role isn't for creating document